### ISSUE TITLE
What is Imbalance Ratio?

### ISSUE THREAD

**Issue by ayushmankd** [adizz2407/Fuzzy-SVM#1]
What is the importance of making imbalance ratio of training and testing same?
How it will affect performance when we don't do this?

**Comment by adityasahugit** [comment]
Hi, Sorry for the late response,
If we have imbalance ration same for train and test set, then we have a less chances of overfitting.
It can affect the perfomance , since more instances of a particular class could be trained for our model and it may decrease the accuracy score for our unseen data.
